src/packages/tokens/infos/commonTokens/sepolia.ts

```diff
@@ -44,10 +45,24 @@ export const tokens: Token[] = [
     symbol: 'crvUSD',
     asset: crvUsdLogo,
   },
+  // TODO: remove duplicate
   {
-    address: '0xdb633c11d3f9e6b8d17ac2c972c9e3b05da59bf9',
+    address: '0x66ebd019E86e0af5f228a0439EBB33f045CBe63E',
     decimals: 18,
     symbol: 'XVS',
     asset: xvsLogo,
   },
+  // TODO: remove duplicate
```

Comment:
Do we have duplicates because we're using some sort of temporary XVS contract?

Reply:
Not really, I made this as a temporary fix to Sepolia reporting two XVS addresses as reward tokens, 0x66ebd019E86e0af5f228a0439EBB33f045CBe63E and 0xDb633C11D3F9E6B8D17aC2c972C9e3B05DA59bF9. It will get cleaned up by the contract team, so I removed the obsolete addres (0xDb633C11D3F9E6B8D17aC2c972C9e3B05DA59bF9)